Learning-Tools-Interoperability.md

```diff
@@ -0,0 +1,67 @@
+# Learning Tools Interoperability (LTI)
+
+MarkUs integrates with other Learning Management Systems via the LTI standard.
+Currently MarkUs supports the following LMS platforms:
+
+- Canvas
+
+## For LMS Administrators
+
+### Installing MarkUs on an LMS
+
+Each LMS implements their own LTI integration process.
+Typically, only administrators can add LTI integrations.
+
+### Canvas
+
+To add MarkUs to a Canvas instance, see their page on
+[configuring LTI keys](https://community.canvaslms.com/t5/Admin-Guide/How-do-I-configure-an-LTI-key-for-an-account/ta-p/140).
+
+MarkUs can be added via a configuration URL. The configuration URL is available at  /lti_deployments/get_canvas_config
+
+## For Instructors
```

Comment:
Everything below here is canvas specific. Add a section for "Canvas" and include these instructions under that section heading. As we add support for other LMS we can add new sections.